src/program.ts

```diff
@@ -58,15 +61,28 @@ program
         options.caps = 'vision';
       }
       const config = await resolveCLIConfig(options);
-      const httpServer = config.server.port !== undefined ? await startHttpServer(config.server) : undefined;
+
```

Comment:
Let's create a extension/main.ts and redirect there so that there was no code from extesion in here.

Reply:
done